src/sentry/snuba/metrics/fields/base.py

```diff
@@ -196,11 +196,11 @@ def _get_entity_of_metric_mri(
         raise InvalidParams
 
     entity_keys_set: frozenset[EntityKey]
-    if use_case_id == UseCaseKey.PERFORMANCE:
+    if use_case_id is UseCaseID.TRANSACTIONS:
```

Comment:
Why somewhere we are using `==` and here `is`? I would keep `==`, what do you think?

Reply:
I guess different people changed it in different ways.
I generally use `==` with the exception of when I check against `None` where I always try to use `is`.

Reply:
Exactly

Reply:
`is UseCaseId.Transaction` is the correct usage according to PEP 8:

> Comparisons to singletons like None should always be done with is or is not, never the equality operators.

https://peps.python.org/pep-0008/#programming-recommendations